src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -103,6 +104,11 @@ public interface IChartEditorPresenter : IPresenter<IChartEditorView>,
       /// </summary>
       void SetDisplayQuantityPathDefinition(Func<DataColumn, PathElements> displayQuantityPathDefinition);
 
+      /// <summary>
+      /// Sets the output mappings of Simulation Outputs and their mapped Observed Data
+      /// </summary>
+      void SetOutputMappingNames(Cache<string, List<string>> outputObservedDataPairs);
```

Comment:
I think it will be much easier to understand if you have a dedicated object for this. You just have a bunch of strings mapped to string and are dealing with this all the time in the code. Encapsulate this properly and it will become much easier to use

Reply:
@georgeDaskalakis It this clear?